packages/shared/src/hooks/useMobileUxExperiment.ts

```diff
@@ -0,0 +1,17 @@
+import { useFeature } from '../components/GrowthBookProvider';
+import { feature } from '../lib/featureManagement';
+import { MobileUXLayout } from '../lib/featureValues';
+import { useViewSize, ViewSize } from './useViewSize';
+
+interface UseStreakExperiment {
+  useNewMobileLayout: boolean;
+}
+
+export const useMobileUxExperiment = (): UseStreakExperiment => {
+  const layout = useFeature(feature.mobileUxLayout);
```

Comment:
Not sure the evaluation is ok here.
This will already assign people to the experiment even if they not mobile.

Reply:
That's why there is also a check below for tablet view size.

Also, this is just for us to quickly rollback this change in case we need to, mobile UX will not actually be an experiment. See this discussion: https://dailydotdev.slack.com/archives/C0650M2KX8E/p1710934118481569

Reply:
Ah completely missed this.
Will add the flag :)